index.js

```diff
@@ -10,10 +10,6 @@ const register = async function(server, options) {
     if (!event.data) {
       return;
     }
-    if (options.clientErrorsToWarnings && event.tags.indexOf('error') !== -1 && event.tags.indexOf('client') !== -1) {
-      event.tags[event.tags.indexOf('error')] = 'warning';
-    }
-    log(event.tags, event.data, { addErrorTagToErrors: !options.clientErrorsToWarnings });
```

Comment:
umm, you removed the whole point of the library...

Reply:
this scares me a lot - means that the tests weren't actually testing anything.

Reply:
Have updated the tests to ensure we are actually passing the info to logr.